axelar-chains-config/info/testnet.json

```diff
@@ -2453,40 +2318,233 @@
+          "versions": [
+            "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be"
+          ],
+          "structs": {
+            "WeightedSigner": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::weighted_signer::WeightedSigner",
+            "Bytes32": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::bytes32::Bytes32",
+            "WeightedSigners": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::weighted_signers::WeightedSigners",
+            "Signature": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::proof::Signature",
+            "Proof": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::proof::Proof",
+            "Message": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::message::Message",
+            "SignersRotated": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::events::SignersRotated",
+            "ChannelCreated": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::events::ChannelCreated",
+            "ChannelDestroyed": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::events::ChannelDestroyed",
+            "ContractCall": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::events::ContractCall",
+            "MessageApproved": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::events::MessageApproved",
+            "MessageExecuted": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::events::MessageExecuted",
+            "AxelarSigners": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::auth::AxelarSigners",
+            "MessageToSign": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::auth::MessageToSign",
+            "Channel": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::channel::Channel",
+            "ApprovedMessage": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::channel::ApprovedMessage",
+            "OwnerCap": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::owner_cap::OwnerCap",
+            "MessageTicket": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::message_ticket::MessageTicket",
+            "MessageStatus": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::message_status::MessageStatus",
+            "Gateway_v0": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::gateway_v0::Gateway_v0",
+            "CommandType": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::gateway_v1::CommandType",
+            "Gateway": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::gateway::Gateway",
+            "Gateway_v1": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be::gateway_v1::Gateway_v1"
```

Comment:
why is `Gateway_v1` included here? Did you use the wrong Sui release/upgradability test release instead of v1.0.1?